### ISSUE TITLE
The result goes beyond the component

### ISSUE THREAD

**Issue by GillesF88** [johnsfdemo/Followers#1]
When we have many followers the result goes beyond the component
![image](https://user-images.githubusercontent.com/58293608/69801753-3ea79680-11d8-11ea-9eb9-47066e951715.png)

**Comment by johnsfdemo** [comment]
@GillesF88 Yeah, I didn't really anticipate lots of followers since my demos typically only require a couple for illustration purposes. A quick fix might be to go to the Followers.cmp (the component), line 20, and change 'multipleRows="false"' to 'multipleRows="true"'. You might also want to try wrapping the <lightning:layout> in a <div class="slds-scrollable_x" ...> as described here: https://www.lightningdesignsystem.com/utilities/scrollable/
